src/LanguageServer.php

```diff
@@ -69,11 +82,18 @@ public function __construct(ProtocolReader $reader, ProtocolWriter $writer)
      */
     public function initialize(string $rootPath, int $processId, ClientCapabilities $capabilities): InitializeResult
```

Comment:
Ok. I didn't know the dispatcher figures out the correct parameter position, cool!